src/Library/demos/Web View/main.json

```
@@ -0,0 +1,7 @@
+{
+  "name": "Web View",
+  "category": "network",
+  "description": "Load and display webpages and HTML",
+  "panels": ["ui", "preview"],
```

`code` is more relevant here

```suggestion
  "panels": ["code", "preview"],
```